### ISSUE TITLE
iOS error related to setting the GL context.

### ISSUE THREAD

**Issue by julapy** [openframeworks/openFrameworks#4435]
since iOS9 ive seen the messages below come up in the console.
and have noticed that fbos appear not to render correctly as a result.
will look into it, but for now just wanted to log it first...

Oct 15 13:02:59  emptyExample[12837] <Error>: CGContextSaveGState: invalid context 0x0. If you want to see the backtrace, please set CG_CONTEXT_SHOW_BACKTRACE environmental variable.
Oct 15 13:02:59  emptyExample[12837] <Error>: CGContextTranslateCTM: invalid context 0x0. If you want to see the backtrace, please set CG_CONTEXT_SHOW_BACKTRACE environmental variable.
Oct 15 13:02:59  emptyExample[12837] <Error>: CGContextRestoreGState: invalid context 0x0. If you want to see the backtrace, please set CG_CONTEXT_SHOW_BACKTRACE environmental variable.
2015-10-15 13:02:59.512 emptyExample[12837:1133746] Creating OpenGL ES1 Renderer

**Comment by danoli3** [comment]
It's a status bar bug.
However you can bypass it by removing "UIViewControllerBasedStatusBarAppearance" from the plist :D

Apple will have to fix it:
https://forums.developer.apple.com/thread/13683

**Comment by julapy** [comment]
ah yeah that fixes those error messages.
wonder if we should remove `UIViewControllerBasedStatusBarAppearance` from the plist?
or wait for apple to fix this in the next release?

**Comment by danoli3** [comment]
Yeah cut it out of the example for now. I think we put it in there to stop iOS 7.0 status bar from being default translucent. However there are other ways of removing that now from Xcode's point of view (outside of that setting).
